mkat_fpga_tests/test_cbf.py

```diff
@@ -479,10 +479,10 @@ def get_actual_phases():
                 # use correlator_fixture.corr_conf[]
                 # correlator_fixture.katcp_rct.req.delays time.time+somethign
                 # See page 22 on ICD ?delays on CBF-CAM ICD
-                reply = correlator_fixture.katcp_rct.req.input_labels()
-                source_name = reply.arguments[1].split()
+                reply, informs = correlator_fixture.katcp_rct.req.input_labels()
+                source_name = informs.arguments[1].split()
```

Comment:
reply - status is concatenated with the messages received. 
informs returns an empty list.